code/modules/client/preferences_savefile.dm

```diff
@@ -134,6 +137,8 @@
 	S["skin_red"]			>> r_skin
 	S["skin_green"]			>> g_skin
 	S["skin_blue"]			>> b_skin
+	S["horn_style_name"]	>> horns
+	S["marking_style_name"]	>> m_style

```

Comment:
don't really need to bother with the savefile at all, it's thoroughly ripped out already and no one else bothered to update it